plugins/CorePluginsAdmin/vue/src/PluginsIntro/PluginsIntro.vue

```diff
@@ -38,15 +44,23 @@ import {
   translate,
   MatomoUrl,
 } from 'CoreHome';
+import InstallAllPaidPluginsButton from '../../../../Marketplace/vue/src/InstallAllPaidPluginsButton/InstallAllPaidPluginsButton.vue';
```

Comment:
@sgiehl Changing it to `import { InstallAllPaidPluginsButton } from 'Marketplace';` gives below error at runtime.

![image](https://github.com/user-attachments/assets/b38afc31-48a5-4d48-be9a-c6cdd317bc3d)

Reply:
I guess that doesn't work due to a circular dependency. The Marketplace component requires CorePluginsAdmin for the field inputs and stuff like that, so you can require the Marketplace within CorePluginsAdmin.
The only solution for this would we to move the component to CorePluginsAdmin. I guess we might even need to do that, as the Marketplace could always be disabled, but the plugins admin should also work without.